connector_prestashop/tests/test_import_partner.py

```diff
@@ -104,7 +104,7 @@ def test_import_partner_category_record(self):
 
         expected = [
             ExpectedCategory(
-                name='Customer A',
+                name='Customer',
```

Comment:
```
FAIL: test_import_partner_category_record (odoo.addons.connector_prestashop.tests.test_import_partner.TestImportPartner)
` Import a partner category
Traceback (most recent call last):
`   File "connector_prestashop/tests/common.py", line 71, in _decorated
`     result = func(self, *args, **kwargs)
`   File "connector_prestashop/tests/test_import_partner.py", line 110, in test_import_partner_category_record
`     self.assert_records(expected, category_bindings)
`   File "connector_prestashop/tests/common.py", line 221, in assert_records
`     '\n'.join(message)
` AssertionError: Records do not match:
`
`  - prestashop.res.partner.category(name: Customer A)
`  + prestashop.res.partner.category(name: Customer)
```

In a fresh PS customer group 3 is named _Customer_ instead of _Customer A_.